runner/runners/invoke.go

```diff
@@ -61,6 +61,7 @@ func (inv *Invoker) Run(cmd *runner.Command, id runner.RunID) (abortCh chan<- st
 func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struct{}, updateCh chan runner.RunStatus) (r runner.RunStatus) {
 	log.Infof("run. id: %v", id)
 	defer func() {
+		inv.stat.Latency(stats.WorkerTaskLatency_ms).Time().Stop()
```

Comment:
this should track how long it takes to do the checkout + run + record stdout/stderr actions.